sympy/polys/ring_series.py

```diff
@@ -1485,19 +1483,19 @@ def rs_cos(p, x, prec):
     if c:
         if R.domain is EX:
             c_expr = c.as_expr()
-            t1, t2 = sin(c_expr), cos(c_expr)
+            _, _ = sin(c_expr), cos(c_expr)
```

Comment:
to indicate what is going on I guess this could be written as `_ok = rhs`

Reply:
Not sure if I follow. (Not even sure that this particular part of the code is needed, the other "useless" calls are withing `try`-`except`, but an exception here will simply break stuff (hence, I doubt that the idea is actually to compute it both here and later as it may just as well break there...)

Reply:
@jksuom might have an idea here.

Reply:
Maybe a different error will arise in the first evaluation and calling code will look for that error. Not sure.